logic/src/main/nl/jft/logic/participant/Participant.java

```diff
@@ -1,8 +1,17 @@
 package nl.jft.logic.participant;
 
 /**
+ * The {@code Participant} interface defines a participant that can partake in {@link nl.jft.logic.match.Match Matches} and {@link nl.jft.logic.tournament.Tournament Tournamements}.
+ * A {@code Participant} is stored in the database and thus should be //TODO add identifiable link when implemented.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1135)